### PR TITLE
Add support for `*.jvm` config files

### PR DESCRIPTION
**Pull request by vutny**

### CODE REVIEW

**Review by olehmd — APPROVED** [review]

File: zoomdata/services/install.sls [hunk 1]
```diff
@@ -252,4 +252,36 @@ zoomdata-user-limits-conf:
...
+    {%- endif %}
+    {%- set jvm_file = salt['file.join'](zoomdata.prefix, srv ~ '.jvm') %}
+
+{{ service }}_jvm:
+  file.managed:
+    - name: {{ jvm_file }}
+    - source: salt://zoomdata/templates/service.jvm
+    - template: jinja
+    - defaults:
+        header: {{ zoomdata.header|default('', true)|yaml() }}
```
Comment (Niyakiy) [inline zoomdata/services/install.sls:270]: Are you sure Zoomdata launcher properly handles comments inside the .jvm files?

File: zoomdata/templates/service.jvm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{%- endif %}
+{%- for opt in options %}
+    {%- if opt.startswith('-') %}
+{{ opt }}
+    {%- else %}
+-{{ opt }}
+    {%- endif %}
+{%- endfor %}
+
+{#- vim: ft=jinja sw=4 sts=4 et
```
Comment (Niyakiy) [inline zoomdata/templates/service.jvm:16]: why do we need editor marks here?

File: zoomdata/services/install.sls (hunk 1 above)
Comment (vutny) [reply zoomdata/services/install.sls:270]: I have tested it.

File: zoomdata/templates/service.jvm (hunk 2 above)
Comment (vutny) [reply zoomdata/templates/service.jvm:16]: Jinja is complex, you should see any issues as soon as you type. All template files already have modelines.

**Review by Niyakiy — APPROVED** [review]

**Pull request merged**